docs/iOSAppium2SetupGuide-README.md

```diff
@@ -12,6 +12,10 @@ Set the **usePrebuiltWDA** capability to false in a capabilities file, you can u
     
     "usePrebuiltWDA": false
 
+Add the capability **iosDeviceType** and set it to real in a capabilities file:
+
+    "iosDeviceType": real
```

Comment:
I don't see this capability in the capabilities file. Can you update it there? Also, is this capability required for local "real" iOS device as well?

Reply:
capabilities are in browserstack caps file, i have updated the sentence specifically for browserstack,
and we dont need this for local execution.